src/mlpack/methods/hmm/hmm_impl.hpp

```diff
@@ -643,23 +654,48 @@ void HMM<Distribution>::Backward(const arma::mat& dataSeq,
   }
 }
 
+template<typename Distribution>
+void HMM<Distribution>::ConvertToLogSpace() const {
```

Comment:
Do you think we can add some comments here (and to the members we added) to clarify what they're for?  And also that they should be removed in mlpack 4.0.0.  (I'll open a follow-up issue when we're done and link it to the mlpack 4.0.0 milestone so we don't forget. :))

Reply:
yes, sure